plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -154,6 +154,17 @@ private static IFile materialize(IProject project, AppEngineProjectConfig config
 
     copyChildFile("favicon.ico", webapp, subMonitor.newChild(5)); //$NON-NLS-1$
 
+    if (config.getAsMavenProject()) {
+      Map<String, String> mavenCoordinates = new HashMap<>();
+      mavenCoordinates.put("projectGroupId", config.getMavenGroupId()); //$NON-NLS-1$
+      mavenCoordinates.put("projectArtifactId", config.getMavenArtifactId()); //$NON-NLS-1$
+      mavenCoordinates.put("projectVersion", config.getMavenVersion()); //$NON-NLS-1$
+      if (!isStandardProject) {
```

Comment:
This if shouldn't be necessary. For now a standard project should return false from getAsMavenProject.